backend/clubs/models.py

```diff
@@ -1895,6 +1896,42 @@ def send_confirmation_email(self):
             )
 
 
+class TicketTransfer(models.Model):
```

Comment:
For consistency's sake, can we call this `TicketTransferRecord`? Goes well with `TicketTransactionRecord`.